package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stepfinance/solana-market-aggregator",
-  "version": "1.3.0",
+  "version": "1.3.2",
```

Comment:
As mentioned [here](https://github.com/step-finance/solana-market-aggregator/pull/10#discussion_r756220108), want to avoid bumping versions inside of PRs and instead use `yarn version` against main.